epubmaker_postprocessing.js

```diff
@@ -7,33 +7,12 @@ fs.readFile(file, function editContent (err, contents) {
           xmlMode: true
         });
 
-// Add links back to TOC to chapter heads
-  $("section[data-type='chapter'] h1").each(function () {
-    var newlink = "<a href='toc01.html'>" + $( this ).text() + "</a>";
-    $(this).empty();
-    $(this).prepend(newlink); 
-  });
-
-// Add links back to TOC to appendix heads
-  $("section[data-type='appendix'] h1").each(function () {
-    var newlink = "<a href='toc01.html'>" + $( this ).text() + "</a>";
-    $(this).empty();
-    $(this).prepend(newlink); 
-  });
-
-// add link back to TOC to preface heads
-  $("section[data-type='preface'] h1").each(function () {
-    var newlink = "<a href='toc01.html'>" + $( this ).text() + "</a>";
-    $(this).empty();
-    $(this).prepend(newlink); 
-  });
-
-// Add links back to TOC to part heads
-  $("div[data-type='part'] h1").each(function () {
-    var newlink = "<a href='toc01.html'>" + $( this ).text() + "</a>";
-    $(this).empty();
-    $(this).prepend(newlink); 
-  });
+// Add links back to TOC to all section heads and part heads
+$("body>section h1, div[data-type='part']>section h1, div[data-type='part'] h1").each(function () {
```

Comment:
This occurred to me, but I figured I'd match what was getting selected before.  I'll do it :)